Library/Formula/rabbitmq-c.rb

```diff
@@ -31,10 +29,10 @@ def install
     args << "-DBUILD_API_DOCS=OFF"
 
     args << if build.with? "tools"
-      "-DBUILD_TOOLS=ON"
-    else
-      "-DBUILD_TOOLS=OFF"
-    end
+              "-DBUILD_TOOLS=ON"
```

Comment:
It's not a mistake. I aligned else with if to fix `brew audit --strict` warning.